docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -179,14 +179,69 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Composants" href="/fr/guides/components/" />
 </CardGrid>
 
+### Encarts
+
+Les encarts (également connus sous le nom de « admonitions ») sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
```

Comment:
```suggestion
Les encarts (également connus sous le nom de « admonitions » ou « asides » en anglais) sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
```

> I never seen "admonitions" in French, I usualy use _Asides_ and not _Encarts_, I searched how it's translated, except for inserts, I haven't really found better words. I accept this edition, but I'm in favour of not wanting to change "Asides". 🤣

Yeah, we are not really used to it but you initially used the proper word. For reference, here is the [Wikipedia reference](https://fr.wikipedia.org/wiki/Mod%C3%A8le:Encart) for their aside style called "encart":

> permet de créer un paragraphe de texte dont le rôle spécifique est signalé par une icône

Altho, there are a few places in the docs where we give both the French and English version when it's not so common (even tho correct), e.g. "logger". What do you think of my suggestion of specifying both versions?

Reply:
> Altho, there are a few places in the docs where we give both the French and English version when it's not so common (even tho correct), e.g. "logger". What do you think of my suggestion of specifying both versions?

I think leave the english version is better to search with keywords. Not for all words, but in this case seems a good option, and also for logger.

Like for the titre of the section : 
> Encarts (Asides)

Or just leave the word in notes like "également connus sous le nom de « admonitions » ou « asides » en anglais" 👍🏼